optimizer-hints.md

```diff
@@ -14,7 +14,7 @@ TiDB supports optimizer hints, which are based on the comment-like syntax introd
 
 ## Syntax
 
-Optimizer hints are specified within `/*+ ... */` comments and follow behind the `SELECT`, `UPDATE` or `DELETE` keyword in a SQL statement. The hint names are case insensitive. If you want to use multiple hints, separate them with commas.
+Optimizer hints are specified within `/*+ ... */` comments and follow behind the `SELECT`, `UPDATE` or `DELETE` keyword in a SQL statement (optimizer hints are not supported behind the `INSERT` keyword). The hint names are case insensitive. If you want to use multiple hints, separate them with commas.
 
 For example, the following query uses three different hints:
```

Comment:
It is easier to read this fact by moving it from the previous paragraph to here.
```suggestion
 Multiple hints can be specified by separating with commas. For example, the following query uses three different hints:
```